acceptance/indy_acceptance/indy-tests/README.md

```diff
@@ -0,0 +1,2 @@
+Note: You need to set the PYTHONPATH before you run a test. It should point to the indy_acceptance folder.
+e.g.: export PYTHONPATH=$PYTHONPATH:/home/user_name/indy_acceptance
```

Comment:
The base directory for indy acceptance tests is `indy-node/acceptance`. It should be added to `PYTHONPATH`, not `indy-node/acceptance/indy_acceptance/indy-tests`. (By the way, in PyCharm the directory being opened as a project is added to `PYTHONPATH` automatically.) `indy_acceptance` is the top-level package which contains all the acceptance stuff. This package contains the subpackage `test` which contains pytest-based tests. This subpackage was created just to follow the common layout for all indy projects where all the pytest-based tests are located in subpackages named `test`. Taking into account that the new acceptance tests are not driven by any unit test framework but are standalone python scripts, we should locate them at `indy_acceptance` package, not at `indy_acceptance/indy-tests`. Since `indy-node/acceptance` will be the content root and `indy_acceptance` package will be the top-level package, then all the imports must be corrected correspondingly. For example, `from libraries import utils` must be changed to `from indy_acceptance.libraries import utils`. (In general, it is a bad practice to use so common names as 
`libraries` for project-specific top-level packages.)

Reply:
Updated. Please refer to the latest code.